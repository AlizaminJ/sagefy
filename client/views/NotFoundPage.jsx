import React from 'react'
import { string } from 'prop-types'
import Layout from './components/Layout'
import Icon from './components/Icon'

export default function NotFoundPage({ hash }) {
  return (
    <Layout
      hash={hash}
      page="NotFoundPage"
      title="Not Found"
      description="Sagefy did not find that page."
    >
      <section>
        <h1>
          I couldn&apos;t find that page <Icon i="error" s="h1" />
        </h1>
        <p>
          <Icon i="error" /> 404 Not Found <Icon i="error" />{' '}
        </p>
        <p>
          <a href="/">
            Go back <Icon i="home" /> home
          </a>
        </p>
      </section>
    </Layout>
  )
}

NotFoundPage.propTypes = { hash: string.isRequired }
