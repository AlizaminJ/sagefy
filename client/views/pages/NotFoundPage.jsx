import React from 'react'
import { Link } from 'react-router-dom'
import Icon from '../components/Icon'

export default function NotFoundPage() {
  return (
    <div className="NotFoundPage">
      <section>
        <h1>
          I couldn&apos;t find that page <Icon i="error" s="xxl" />
        </h1>
        <p>
          <Icon i="error" /> 404 Not Found <Icon i="error" />{' '}
        </p>
        <p>
          <Link to="/">
            Go back <Icon i="home" /> home
          </Link>
        </p>
      </section>
    </div>
  )
}
