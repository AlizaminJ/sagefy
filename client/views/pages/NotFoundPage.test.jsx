import NotFoundPage from './NotFoundPage'

describe('NotFoundPage', () => {
  it('should render the not found page', () => {
    expect(NotFoundPage()).toMatchSnapshot()
  })
})
