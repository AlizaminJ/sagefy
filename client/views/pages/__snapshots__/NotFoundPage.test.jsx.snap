// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should render the not found page 1`] = `
<div
  className="NotFoundPage"
>
  <h1>
    I couldn't find that page 
    <Icon
      i="error"
      s="xxl"
    />
  </h1>
  <p>
    <Icon
      i="error"
      s="m"
    />
     404 Not Found 
    <Icon
      i="error"
      s="m"
    />
     
  </p>
  <p>
    <Link
      replace={false}
      to="/"
    >
      Go back 
      <Icon
        i="home"
        s="m"
      />
       home
    </Link>
  </p>
</div>
`;
